magpie/db.py

```diff
@@ -14,6 +14,7 @@
 import warnings
 import logging
 import time
+import six
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [third party import "import six" should be placed before "from magpie.constants import get_constant"](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)